File: benchmarks/net/benchmark1/src/models/accesscode_eto.rs
use chrono::{DateTime, NaiveDateTime, Utc};
use serde::{Serialize, Deserialize};

#[derive(Serialize, Deserialize)]
#[serde(rename_all="camelCase")]
pub struct AccessCodeEto {
    pub id: Option<i64>,
    pub ticket_number: Option<String>,
    pub creation_time: Option<DateTime<Utc>>,
    pub start_time: Option<DateTime<Utc>>,
    pub end_time: Option<DateTime<Utc>>,
    pub queue_id: i64,
    pub visitor_id: i64,
}

File: net/source/Templates/WebAPI/Devon4Net.WebAPI.Implementation/Business/AccessCodeManagement/ETO/AccesscodeETO.cs
using Devon4Net.WebAPI.Implementation.Business.QueueManagement.ETO;
using Devon4Net.WebAPI.Implementation.Business.VisitorManagement.ETO;
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Devon4Net.WebAPI.Implementation.Business.AccessCodeManagement.ETO
{
    public partial class AccesscodeETO
    {
        
        //public int modificationCounter { get; set; }
        public long id { get; set; }
        public string ticketNumber { get; set; }
        public DateTime creationTime { get; set; }
        public DateTime? startTime { get; set; }
        public DateTime? endTime { get; set; }
        public long queueId { get; set; }
        public long visitorId { get; set; }
        
        
        
    }
}
